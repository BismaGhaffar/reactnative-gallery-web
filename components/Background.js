import styled from 'styled-components'

const Background = styled.div`
  background-image: url('/static/images/background_blur.jpeg');
  background-attachment: scroll;
  background-position: 50% 50%;
  background-size: cover;
  background-repeat: no-repeat;
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  content: '';
  z-index: 0;
`

export default Background
