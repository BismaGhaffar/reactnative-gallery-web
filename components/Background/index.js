import styled from 'styled-components'

const Background = styled.div`
  background-image: url('/static/images/background.jpeg');
  background-attachment: scroll;
  background-position: 50% 50%;
  background-size: cover;
  background-repeat: no-repeat;
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  content: '';
  z-index: 0;
  -webkit-filter: blur(3px);
  filter: blur(3px);
`

export default Background
