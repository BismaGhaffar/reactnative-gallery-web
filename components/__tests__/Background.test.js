import React from 'react'
import renderer from 'react-test-renderer'
import Background from '../Background'

it('Background can be created', () => {
  const background = renderer.create(<Background />)

  expect(background).toBeDefined()
  const tree = renderer.create(<Background />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('<Background /> toMatchSnapshot', () => {
  const tree = renderer.create(<Background />).toJSON()
  expect(tree).toMatchSnapshot()
})
