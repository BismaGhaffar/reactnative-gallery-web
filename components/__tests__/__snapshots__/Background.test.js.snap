// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> toMatchSnapshot 1`] = `
<div
  className="Background-s1pndi9l-0 izkYGM"
/>
`;

exports[`Background can be created 1`] = `
<div
  className="Background-s1pndi9l-0 izkYGM"
/>
`;
